crates/rust-analyzer/tests/slow-tests/tidy.rs

```diff
@@ -280,6 +280,7 @@ fn check_todo(path: &Path, text: &str) {
         "ast/make.rs",
         // The documentation in string literals may contain anything for its own purposes
         "ide_db/src/helpers/generated_lints.rs",
+        "ide_assists/src/utils/gen_trait_body.rs",
```

Comment:
Nit: whitespace

Reply:
@lnicola Can you elaborate? Do you mean you want me to add a newline?

Reply:
Sorry, you can ignore that. The GitHub app shows the other lines starting from the first column and this one as indented, but it looks fine in browser.